podium-common/src/main/java/nl/thehyve/podium/common/config/MapperConfiguration.java

```diff
@@ -11,6 +11,7 @@
     @Bean
     public ObjectMapper objectMapper() {
         ObjectMapper mapper = new ObjectMapper();
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
Why? I expect clients to send valid messages and consider unknown properties a mistake on the client side (perhaps a typo), of which the client should be informed.

Reply:
Because e.g. `RequestFormComponent` modifies a `Request` object, adding at least one new field (`hasAttachmentsTypes`) — and sends the modified object to an API endpoint.

I agree that it is better to keep the API a bit more strict and to refactor the FE instead to exclude unnecessary fields from requests, but I'd like to keep it lenient for now.

Reply:
Makes sense.